python_scripts/linear_models_sol_01.py

```diff
@@ -46,9 +46,12 @@ def linear_model_flipper_mass(
 # %% [markdown]
 # ## Main exercise
 #
-# Given a vector of the flipper length, several weights and intercepts to
-# plot several linear model that could fit our data. Use the above
-# helper function to visualize both the model and data.
+# Define a vector ```weights = [...]``` and a vector ```intercepts = [...]``` of
+# the same length. Each pair of entries ```(weights[i], intercepts[i])``` tags a
+# different model. Use these vectors along with the vector
+# ```flipper_length_range``` to plot several linear models that could possibly
+# fit our data. Use the above helper function to visualize both the models and
+# the real samples.
```

Comment:
is it normal to have three backticks to define a text as code here?

Reply:
I had in mind that three back-ticks are good for quoting code in a particular format, though this is probably unnecessary here.

Reply:
Triple backticks are probably only useful for multi-line code blocks.